rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -659,6 +661,10 @@ error_code sys_usbd_receive_event(ppu_thread& ppu, u32 handle, vm::ptr<u64> arg1
 	*arg1 = ppu.gpr[4];
 	*arg2 = ppu.gpr[5];
 	*arg3 = ppu.gpr[6];
+
+	if (*arg1 == SYS_USBD_ATTACH)
+		lv2_obj::wait_timeout(5000);
```

Comment:
```suggestion
		lv2_obj::sleep(ppu), lv2_obj::wait_timeout(5000);
```
waiting might not have any effect without ppu sleep in some situations.
add it to the above line and the first hack.